tests/testUtils.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const {
+  inputMeta,
+  inputValidation,
+  inputMapping,
+  inputConstants
+} = require('../src/constants')
+
+const pathToDirectory = path.resolve('endpoints', 'integrationTest')
+const metaFilePath = path.resolve(pathToDirectory, inputMeta)
+const inputValidationFilePath = path.resolve(pathToDirectory, inputValidation)
+const inputMappingFilePath = path.resolve(pathToDirectory, inputMapping)
+const inputConstantsFilePath = path.resolve(pathToDirectory, inputConstants)
+
+const metaFileContent = {
+  name: 'IntegrationTest',
+  endpoint: {
+    pattern: '/integrationTest'
+  },
+  transformation: {
+    input: 'XML',
+    output: 'JSON'
+  }
+}
+
+const inputValidationFileContent = {
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string'
+    },
+    surname: {
+      type: 'string'
+    },
+    attributes: {
+      type: 'array',
+      items: {
+        type: 'string'
+      }
+    }
+  },
+  required: ['name']
+}
```

Comment:
I think this is okay for now, but would be great if we could have these defined within the test itself, so that it might be possible to have variations of this per test being run. This can be a future refactor